### PR TITLE
fix(kousa): add .envrc.bat for Windows.

### PR DESCRIPTION
**Pull request by samarmohan**
Some people are on Windows... Maintainers can change the name of the file if they want.

### CODE REVIEW

File: kousa/.envrc.bat [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+set DISCORD_CLIENT_ID=ntahunhaoneuhtohauehonu
+set DISCORD_CLIENT_SECRET=aoneunthaehouh
+
+set GOOGLE_CLIENT_ID=ntahunhaoneuhtohauehonu
+set GOOGLE_CLIENT_SECRET=aoneunthaehouh
+
+set TWITTER_BEARER_TOKEN=aoidjwqdjqiowdjwqadjoqwidoqw
+
+set DATABASE_URL=postgres://asdouaoisdwionasa
```
Comment (amitojsingh366) [inline kousa/.envrc.bat]: this is not what we want, the env variables shown in CONTRIBUTING.md are just examples and need to be filled manually... we don't want to supply the contributors with our own versions of them.

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

File: kousa/.envrc.bat (hunk 1 above)
Comment (samarmohan) [reply kousa/.envrc.bat]: So what do I add?? Do we want the actual versions, blank versions, or filler ones.

Comment (benawad) [inline kousa/.envrc.bat]: this is fine, just change this to a legit postgres url pointing at localhost with the database name `kousa_repo2`

**Review by benawad — CHANGES_REQUESTED** [review]

**Comment by goldyydev** [comment]
I think that .envrc and .envrc.bat should be ignored to avoid users leaking their personal info (with shell plugins like direnv who need it to function).

**Comment by amitojsingh366** [comment]
> I think that .envrc and .envrc.bat should be ignored to avoid users leaking their personal info (with shell plugins like direnv who need it to function).

i agree, they should just copy the template from CONTRIBUTING.md into their own .envrc

**Comment by samarmohan** [comment]
Should I just remove .envrc in this PR? I’ll change the goal of it but it will help.

File: kousa/.envrc.bat (hunk 1 above)
Comment (samarmohan) [reply kousa/.envrc.bat]: We can just add that in the template env file.

**Comment by benawad** [comment]
I want it to work out of the box with .envrc

**Pull request closed without merging**